pandas/io/formats/printing.py

```diff
@@ -501,16 +508,20 @@ def format_object_attrs(obj, include_dtype=True):
 
     Returns
     -------
-    list
+    list of 2-tuple
 
     """
-    attrs = []
+    attrs = []  # type: List[Tuple[str, Union[str, int]]]
     if hasattr(obj, "dtype") and include_dtype:
-        attrs.append(("dtype", "'{}'".format(obj.dtype)))
+        # error: "Sequence[Any]" has no attribute "dtype"
```

Comment:
Hmm this type of introspection is going to be interesting with annotations. Something to think about moving forward

Reply:
see https://github.com/python/mypy/issues/1424